cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/ConsentRespExt.kt

```diff
@@ -76,9 +76,9 @@ internal fun Map<String, Any?>.toCCPAUserConsent(uuid: String?, applies: Boolean
             val status = CcpaStatus.values().find {
                 it.name == s
             }
-            return@let status ?: CcpaStatus.unknown
+            status ?: CcpaStatus.unknown
```

Comment:
Let’s also verify if this extension is currently used in the code and if the functions that use it are also used. If they are belonging to the v6 and are not used in the v7 we can delete them

Reply:
@carmelo-iriti this function is not used in v7 at all. It is used in v6 only.

Reply:
In this case I would say that we can remove it and its all its flow

Reply:
Wait a minute. It is used while changing `localStateStatus`